src/NUnitTestAdapter/NUnit3TestExecutor.cs

```diff
@@ -322,9 +322,9 @@ private void RunAssembly(string assemblyPath, IGrouping<string, TestCase> testCa
             }
             finally
             {
-#if NET35
+//#if NET35
                 dumpXml?.Dump4Execution();
-#endif
+//#endif
```

Comment:
Do you think these lines will be needed in the future?

Reply:
Gone and Done